docs/source/operations/setting_dlt.md

```diff
@@ -31,6 +31,12 @@ For a successful setup of DLT Network all the pods should be in running state.
 
 
 ## Deleting an existing DLT network
+The above mentioned playbook [site.yaml](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/shared/configuration/site.yaml) ([ReadMe](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/shared/configuration/README.md)) can be manually run after the configuration file [network.yaml](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml), [network yaml details for R3-Corda](corda_networkyaml.md) and [network yaml details for Hyperledger-Fabric](fabric_networkyaml.md), has the required specifications which was used to setup the network by using the command:
```

Comment:
"The playbook site.yaml ..."
Please remove references to "manually" in this page, and rephrase the sentences. 
Instead of providing all network.yaml, say "the network configuration file of the chosen platform (for example ..) " And give only one example file link.